frontend/app/src/locales/en.json

```diff
@@ -934,6 +934,10 @@
       "title": "Icon upload",
       "description": "The icon upload failed: {message}"
     },
+    "refresh_icon": {
+      "title": "Refresh icon",
```

Comment:
Maybe this could be something that explains better the functionality? What about `Restore icon`?

Reply:
`restore` seems not to be the right word. 
What about `Get the latest icon`?

Reply:
That sounds better :)